torch/lib/c10d/FileStore.cpp

```diff
@@ -144,7 +194,7 @@ class File {
     while (count > 0) {
       auto rv = syscall(std::bind(::write, fd_, buf, count));
       SYSASSERT(rv, "write");
-      buf = (uint8_t*)buf + count;
+      buf = (uint8_t*)buf + rv;
```

Comment:
this should have already landed in master, you might need a rebase.

Reply:
yes, will do it.